Algorithms/Greedy/CoinProblem.java

```diff
@@ -0,0 +1,219 @@
+
+//		Coin Problem Description -
+//		Given a value V, if we want to make change for the amount V and
+//		we have infinite supply of each of the denominations in some
+//		currency say {1,2,5,10,20,50} (which will be taken as input from the user}
+//		valued coins/notes, what is the minimum number of coins and/or notes
+//		needed to make the change?
+//
+//		Author - Aastha Aneja (Github handle - Aashu24)
+//		Email - anejaaastha@gmail.com
+		
+import java.util.Arrays;
+import java.util.Scanner;
+
+public class CoinProblem {
+
+	public static void main(String[] args) {
+
+		// Scanner declaration to take input
+		Scanner in = new Scanner(System.in);
+
+		// This variable is for the size of coin array
```

Comment:
Add prompts for the user to take input e.g. `Please enter the amount for which change is to be made:`
Do this everywhere input is being taken from the user.